chimp.md

```diff
@@ -0,0 +1,213 @@
+- Articles on Zero-Knowledge Proof (ZKP) and Multi-Party Computation (MPC) have discussed the potential for private transactions, which further supported Chimp’s decision to integrate privacy technology into its DEX.
+
+Currently, there are no direct competitors within the Polkadot/Substrate/Kusama ecosystem that focus specifically on providing a private, anonymous DEX solution.
+
+How is your project different?
+- Unique Privacy Focus: While other DEXes or DeFi platforms within the ecosystem offer basic decentralized trading, none have integrated end-to-end encryption or the ability to execute fully anonymous trades. Chimp Exchange is distinct in its approach by focusing on E2EE, hidden liquidity pools, and anonymous fundraising—features that are currently missing in other platforms.
+- Integration with Polkadot: The Polkadot ecosystem’s focus on interoperability aligns perfectly with Chimp Exchange’s cross-chain trading functionality. However, Chimp goes further by ensuring that cross-chain trades are conducted privately, a feature not currently offered by other projects in the ecosystem.
+
+Why might such a project not have been possible, successful, or attempted?
+- Technical Complexity: Implementing privacy layers and E2EE in a decentralized environment is highly complex, requiring advanced cryptography, ZKPs, and a strong understanding of security vulnerabilities. Many projects in the DeFi space focus on public transparency, making it difficult to shift toward privacy-focused solutions.
+- Regulatory Concerns: Privacy solutions have often faced challenges with regulatory compliance, as governments and institutions require transparency for anti-money laundering (AML) and know your customer (KYC) protocols. Chimp Exchange balances privacy and compliance, focusing on anonymizing transactions without enabling illicit activities.
+
+Are there any projects similar to yours in related ecosystems?
+
+Yes, similar projects exist in other ecosystems, including:
+- ZKSwap (Ethereum-based): A Layer 2 DEX that focuses on privacy but doesn’t offer end-to-end encryption or cross-chain capabilities like Chimp Exchange.
+- Tornado Cash (Ethereum-based): A mixing protocol that offers transaction anonymity but lacks a fully integrated trading platform like Chimp Exchange.
+- SecretSwap (Secret Network): This DEX offers privacy but is limited to the Secret Network and doesn’t provide the cross-chain functionality that Chimp Exchange is building.
+
+How is Chimp Exchange different?
+- Cross-Chain Privacy Trading: Unlike ZKSwap or SecretSwap, Chimp Exchange is focused on enabling private trading across multiple chains via the Ten Protocol. This cross-chain compatibility, coupled with complete anonymity, sets Chimp apart.
+- End-to-End Encryption: While Tornado Cash and ZKSwap offer forms of transaction mixing, Chimp Exchange integrates full E2EE, ensuring that trades are not only private but also encrypted at every stage.
```

Comment:
Could you please elaborate on this point? What data is subject to e2e encryption in your design and how would it work?